resources/charlie/chainspec.toml

```diff
@@ -44,7 +44,10 @@ entropy_duration_millis = 10800000
 # Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
 # 172800000 ms = 2 days.
 voting_period_duration_millis = 172800000
-# Integer between 0 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
+# Integer between 1 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
+# It is the percentage of validators that would need to be malicious to make two honest nodes see two conflicting
```

Comment:
I think we're using _malicious_ in places where it means behavior other than equivocating. Can we be more explicit here?